tldr.py

```diff
@@ -349,7 +349,7 @@ def main():
                 for command in rest.command:
                     result = get_page(command, platform=options.os, remote=options.source)
                     if not result:
-                        print((
```

Comment:
This is actually a change in behavior I believe. The print statement will print but continue the loop. Changing it to `sys.exit` will directly exit the program.

Reply:
I see, the program can be called with multiple commands at a time - I wasn't aware of that.

Personally, I don't see much use in that feature - especially since it isn't documented anywhere and other implementations (that I know) don't have it.

If you prefer to keep it, I will fix the PR tomorrow to exit `1` only at the end if any errors were caught.  
If not, I can modify the parsing logic to get rid of the loops.

Reply:
Yes, let's just `print()` in this case. Others seem fine. I think it may be a breaking change if you remove the loop logic. We just don't know who is using this feature out in the wild.